services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -146,31 +142,30 @@ async def _reset_on_error(
         raise
 
 
+async def task_pull_user_servcices_docker_images(
+    progress: TaskProgress, shared_store: SharedStore, app: FastAPI
+) -> None:
+    assert shared_store.compose_spec  # nosec
+
+    progress.update(message="started pulling user services", percent=ProgressPercent(0))
+
+    await docker_compose_pull(app, shared_store.compose_spec)
+
+    progress.update(
+        message="finished pulling user services", percent=ProgressPercent(1)
```

Comment:
the name `ProgressPercent` suggest that you are now at 1%

Reply:
Yes and it's such a mess to fix now. It touches a lot of systems. Will not rename now.